plugins/osdn/registry.go

```diff
@@ -666,3 +671,39 @@ EndpointLoop:
 
 	registry.baseEndpointsHandler.OnEndpointsUpdate(filteredEndpoints)
 }
+
+func (registry *Registry) trackPod(pod *kapi.Pod) {
+	if pod.Status.PodIP != "" {
```

Comment:
could simplify this by just

    if pod.Status.PodIP == "" {
        return
    }